src/libipc/ipcd/ipcd.c

```diff
@@ -212,6 +216,9 @@ private void * start_accepting()
         perror("pthread_detach() failed");
         pthread_exit(NULL);
       }
+
+      // Check if currently unit testing 
+      if(debug) break;
```

Comment:
This should be an `#ifdef` for consistancy

Reply:
yes you're right! my bad
```suggestion
      #ifdef DEBUG
        break;
      #endif
```